Makefile

```diff
@@ -12,52 +12,91 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
+# Deliverables path
 DESTDIR ?= /usr/local
-
-VERSION=$(shell git describe --match 'v[0-9]*' --dirty='.m' --always --tags)
+BINDIR ?= $(DESTDIR)/bin
+
+# Tools path
+RM ?= /bin/rm
```

Comment:
Isn't this implied by default?
https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html

Reply:
Yeah, will fix it.